src/pages/index.ejs

```diff
@@ -217,29 +217,34 @@ Features are implemented as preventing arming when the drone isn’t leveled and
 											</div>
 										</div>		
 										<div class="single-offer d-flex flex-row pb-30">
-												<div class="icon">
-													<i class="fa fa-chart-area fa-2x"></i>
-												</div>
-												<div class="desc">
-													<a href="#"><h4 class="text-white">BlackBox Viewer</h4></a>
-													<p>Viewer Releases are here: https://github.com/betaflight/blackbox-log-viewer/releases
-		
-															The Latest Viewer source is here:
-														</p>	<p>
-															https://github.com/betaflight/blackbox-log-viewer
-															See BB Logging and Usage Wiki page on using the BB logger.</p>
-												</div>
-											</div>
+					                        <div class="icon">
+                                                <i class="fa fa-chart-area fa-2x"></i>
+                                            </div>
+                                            <div class="desc">
+                                                <h4 class="text-white">BlackBox Viewer</h4>
+                                                <p>
+                                                    Viewer Releases are here:
+                                                </p><p>
+                                                        <a href="https://github.com/betaflight/blackbox-log-viewer/releases">Latest viewer releases</a>
+                                                </p><p>
+                                                        The Latest Viewer source is here:
+                                                </p><p>
+                                                        <a href="https://github.com/betaflight/blackbox-log-viewer">Latest viewer source</a>
+                                                </p><p>
+                                                        See BB Logging and Usage Wiki page on using the BB logger.
+                                                </p>
+                                            </div>
+                                        </div>
 								</div>
 							<div class="col-lg-6">
 									<div class="single-offer d-flex flex-row pb-30">
 										<div class="icon">
 											<i class="fa fa-folder fa-3x"></i>
 										</div>
 										<div class="desc">
-											<a href="#"><h4 class="text-white">Betaflight Releases</h4></a>
+											<h4 class="text-white">Betaflight Releases</h4>
 											<p>
-													Releases can be found on <a href="https://github.com/betaflight/betaflight/releases">github.com/betaflight</a> or downloaded from within the firmware section within the configurator. Also check the Upgrading List to the Right for Release Notes and other Details on the various Versions.
+													Releases can be found in the link bellow or downloaded from within the firmware section within the configurator tool. Also check the Upgrading List to the Right for Release Notes and other Details on the various Versions.
```

Comment:
I'd put this as 'Releases can be found by following the link below.'

Also, we normally refer to configurator as 'Betaflight Configurator', and not as 'configurator tool'.

Reply:
@mikeller thanks for the correction, it seems that sometimes we use different names for the same thing, fixed now